interface-definitions/container.xml.in

```diff
@@ -104,11 +104,26 @@
               </leafNode>
             </children>
           </tagNode>
+          <leafNode name="entrypoint">
+            <properties>
+              <help>entrypoints</help>
```

Comment:
There should be uppercase for first letter in help 
+ describe more detailed for Entrypoint

Reply:
I've published an updated version with help message from man page of podman-run.
I think it should be okay now.